### ISSUE TITLE
Job output pages

### ISSUE THREAD

**Issue by Peter9192** [i-VRESSE/bartended-haddock3#17]
Display haddock output in a nice way.

```
/jobs  --> overview of all jobs
/jobs/jobid --> status of job
/jobs/jobid/result --> overview of outputs
/jobs/jobid/files/... --> nice plots from Sarah
```

**Comment by sverhoeven** [comment]
If you run https://github.com/i-VRESSE/haddock3/blob/create_report/examples/docking-protein-protein/docking-protein-protein-test.cfg as `haddock3 docking-protein-protein-test.cfg` and then ` haddock3-analyse -r run1-test -m 2 5 7`.

You will get `run1-test/analysis/*_caprieval_analysis/report.html` files.

It would be nice on `/jobs/jobid/result` route to list all the available report.html files.

ps. a report.html looks similar to https://wenmr.science.uu.nl/haddock2.4/run/4242424242/195967-E2A-HPR

**Comment by Peter9192** [comment]
For now pick `output/log` file as it is the only file that we know for certain will be there

**Comment by sverhoeven** [comment]
As of #23 and https://github.com/i-VRESSE/bartender/pull/59 you can use JobApi.retrieveJobDirectoriesApiJobJobidDirectoriesGet to list directories.

**Comment by sverhoeven** [comment]
Fixed in #33
